### ISSUE TITLE
Fatal error after adding Container in v9

### ISSUE THREAD

**Issue by rintisch** [b13/container#73]
**Error:**
In the backend (page module) I got a error message after adding a container and saving it.

_**Fatal error:** Access level to B13\Container\View\ContainerLayoutView::$nextThree must be public (as in class TYPO3\CMS\Backend\View\PageLayoutView) in **/var/www/html/public/typo3conf/ext/container/Classes/View/ContainerLayoutView.php** on line **0**_

Steps to reproduce:
- use TYPO3-Version: 9.5.19
- add following in `Configuration/TCA/Overrides/tt_content.php`

```
\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\B13\Container\Tca\Registry::class)->addContainer(
    '2col-container', 
    '2 Column Container',
    'Insert an element dividing the content area into two columns',
    [
        [
            ['name' => 'Left Column', 'colPos' => 201],
            ['name' => 'Right Column', 'colPos' => 202]
        ]
    ] 
```

- add this container in the backend & save
- see error

**Screenshot**:
![grafik](https://user-images.githubusercontent.com/44193208/88388707-565e0b80-cdb5-11ea-8749-1967ac1602f0.png)

**Comment by achimfritz** [comment]
the bug sould be fixed in 1.1.1 . What Version do you have?

**Comment by rintisch** [comment]
Oh, that was a missing information in my bug report, sorry.
Yep, it's fixed with 1.1.1 - `composer req b13/container` somehow installed 1.1.0

Thanks for the quick response!
